docs/tutorials/abn/abn.md

```diff
@@ -34,6 +34,23 @@ Example frontends are provided in *Node.js*, *Python* and *Go*.
 ???+ warning "Before you begin"
     1. Try [your first experiment](../../getting-started/your-first-experiment.md). Understand the main [concepts](../../getting-started/concepts.md) behind Iter8 experiments.
  
+## Launch Iter8 A/B/n service
+
+If not already deployed, deploy the Iter8 A/B/n service. When deploying the service, specify which Kubernetes resources to watch for each application. To watch for versions of the _backend_ application in the _default_ namespace, watch for service and deployment resources:
```

Comment:
```suggestion
Deploy the Iter8 A/B/n service. When deploying the service, specify which Kubernetes resources to watch for each application. To watch for versions of the _backend_ application in the _default_ namespace, configure the service to watch for service and deployment resources:
```

Seems strange to me to start with "if not already deployed" in a tutorial. Also, clarify the last sentence. I had to reread it a few times.

Reply:
I see you refer to the backend application and default namespace as `*backend*` and `*default*` whereas here it is `_backend_` and `_default_`